runtime/ai/analyst_agent.go

```diff
@@ -78,18 +78,20 @@ func (t *AnalystAgent) Handler(ctx context.Context, args *AnalystAgentArgs) (*An
 
 	// If a specific dashboard is being explored, we pre-invoke some relevant tool calls for that dashboard.
 	var metricsViewName string
-	if args.Explore != "" {
-		_, metricsView, err := t.getValidExploreAndMetricsView(ctx, args.Explore)
+	if args.Context.Explore != "" {
```

Comment:
FYI: See the change to this line in this PR: https://github.com/rilldata/rill/pull/8194

Not a blocker for this PR, I can handle this later, but we need to add handling for removing/changing the seeded tool calls if the explore changes later in the conversation (e.g. if they move from one dashboard to another)